evals/solvers/providers/fixie/local_gpu_solver.py

```diff
@@ -28,35 +32,33 @@ class LocalGPUSolver(Solver):
 
     def __init__(
         self,
-        completion_fn_options: Dict[str, Any] = {},
+        model: str,
+        num_gpus: int = torch.cuda.device_count(),
+        extra_options: Optional[Dict[str, Any]] = None,
```

Comment:
The indirection of `completion_fn_options` made it hard to set model name through `completion_args`. Now I can just do `--completion_args=model=test`.